charts/postgres-operator/templates/deployment.yaml

```diff
@@ -54,7 +54,7 @@ spec:
           value: {{ template "postgres-operator.controllerID" . }}
       {{- end }}
       {{- if .Values.extraEnvs }}
-      {{- .Values.extraEnvs | toYaml | nindent 8 }}
+{{ toYaml .Values.extraEnvs | nindent 8 }}
```

Comment:
why is it nindent and not just indent?

Reply:
I missed that the comparable lines in the same file all used indent. I've adjusted the line now to use indent, too.